baseplate/clients/requests.py

```diff
@@ -101,6 +107,35 @@ def http_adapter_from_config(
     return ValidatingHTTPAdapter(**kwargs)
 
 
+PROM_NAMESPACE = "http_client"
+HTTP_LABELS_COMMON = [
+    "http_method",
+    "http_client_name",
+]
+HTTP_LABELS_TERMINAL = [*HTTP_LABELS_COMMON, "http_success"]
```

Comment:
I'm surprised that http_sucess got missed in the spec as well

Reply:
It looks like we're still missing 'http_endpoint'.